### ISSUE TITLE
PC Importer v0.7 with Foundry 9 fails silently: creates the skeleton PC with stats and XP but nothing else

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/pc-importer#3]
Probably a missed deprecation that worked in Foundry 0.8 and was silently removed in Foundry 9

**Comment by spetzel2020** [comment]
Partially fixed in 0.8.0a; still working on remaining fixes.
(As suspected, was missing deprecations for `getEntity` and `createEmbeddedEntities` in `Actor5eFromExt.js#matchForItemType()`)

**Comment by spetzel2020** [comment]
Still not working: Level is being extracted correctly but not set in Class

**Comment by spetzel2020** [comment]
Have traced the problem to the following, as described in my post on #dev-support:

```
 in using createEmbeddedDocuments to directly set the subclass and level for an Actor. I have created the appropriate Item5e with class, subclass, and level set (say for example to "Wizard", "School of Evocation", and 11).
[3:23 PM] Spetzel: I call actor.createEmbeddedDocuments("Item", allItemsData) where allItemsData contains that one class Item. It eventually burrows down to these calls:
_createEmbeddedDocuments()
_preCreateDocumentArray()
toObject()
 
[3:25 PM] Spetzel: In toObject(source=true) as far as i can tell, the data is not copied to an object vanilla, but rather since source=true (the default), it uses predefined default values for this sub-object of class, which overwrites level to 1, and subclass to "".
[3:26 PM] Spetzel: Unfortunately in _preCreateDocumentArray() this is the calling line:
if ( d instanceof foundry.abstract.DocumentData ) d = d.toObject();
 
[3:26 PM] Spetzel: so I can't even pass a parameter to override the default source=true in toObject()
[3:27 PM] Spetzel: This used to work fine for me in Foundry 0.8, and I am thinking that I must be approaching this wrong. Is there another approach or some way I can work around this problem?
```

**Comment by spetzel2020** [comment]
Fixed and Released in 0.8.0
